CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - ubi-rust-builder: Bump Rust toolchain to 1.85.0, cargo-cyclonedx to 0.5.7, and cargo-auditable to 0.6.6 ([#1050]).
+- ubi-rust-builder: Bump base image and update protoc to `30.2` ([#1091]).
```

Comment:
suggestion: Mention the exact version of the ubi-rust-builder image.

```suggestion
- ubi9-rust-builder: Bump base image and update protoc to `30.2` ([#1091]).
```

Reply:
Oops. Yeah done in #89ab14e